SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -235,6 +236,9 @@ public void onMessage(final SwrveConversation conversation) {
                                 intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                                 intent.putExtra("conversation", conversation);
                                 ctx.startActivity(intent);
+
+                                // Report that the message was shown to users
+                                conversation.getCampaign().messageWasShownToUser();
```

Comment:
We send impression events at the moment for conversations on a per page basis (I think)

How will this extra impression event affect reporting?